libs/smart-ngrx/src/socket/delete-entity.function.ts

```
@@ -22,4 +23,15 @@ export function deleteEntity(feature: string, entity: string, ids: string[]) {
       removeIdFromParents(childDefinition, id, parentInfo);
     });
   });
+
+  // Update the virtual array without refreshing the screen
+  const actionService = actionServiceRegistry(feature, entity);
+  actionService.entities.pipe(take(1)).subscribe((entities) => {
+    forNext(ids, (id) => {
+      const entity = entities[id];
+      if (entity && entity.virtualChildren) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)